base/logging.go

```diff
@@ -421,6 +91,12 @@ func Tracef(logKey LogKey, format string, args ...interface{}) {
 	logTo(LevelTrace, logKey, format, args...)
 }
 
+func Statsf(stats []byte) {
+	if statsLogger != nil {
+		statsLogger.logf("%s", string(stats))
```

Comment:
The caller of this is converting a string into `[]byte` just to pass through, and then here we turn it back into a string. Seems like unnecessary work that can be avoided by changing this function signature.

I think also calling: `statsLogger.logf(stats)` is more efficient, as it doesn't need to do as much work to build the string as `%s <- stats` causes.

Reply:
Agreed.  Removed byte slice to string RT, and removed extraneous `%s`